scripts/shared/clusters.sh

```diff
@@ -11,6 +11,7 @@ DEFINE_boolean 'globalnet' false "Deploy with operlapping CIDRs (set to 'true' t
 DEFINE_boolean 'registry_inmemory' true "Run local registry in memory to speed up the image loading."
 DEFINE_string 'cluster_settings' '' "Settings file to customize cluster deployments"
 DEFINE_string 'timeout' '5m' "Timeout flag to pass to kubectl when waiting (e.g. 30s)"
+DEFINE_boolean 'ovn' false "Deploy with OVN as CNI"
```

Comment:
This goes against what we currently have and rolls us back to the days of specifying CNI on the command line.

Why not specify this in cluster_settings?

Reply:
> This goes against what we currently have and rolls us back to the days of specifying CNI on the command line.
> 
> Why not specify this in cluster_settings?

Will it be possible to add CNI to job matrix if we use cluster_settings? I yes, then we don't need extra flag.